assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -37,20 +37,23 @@
    ]
```

Comment:
<p>where you find this code snipet?</p><p> </p><p>I don't think it is something better than the original assignment code. Let's keep the original one and find something else here. Please have a look at the fib() below. We should have some more difficult code here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>